samples/cpp/benchmark_app/benchmark_app.hpp

```diff
@@ -14,15 +14,6 @@
 
 #include "gflags/gflags.h"
 
-// gflags supports uint32 starting from v2.2 only
-#ifndef DEFINE_uint32
-#    ifdef GFLAGS_NAMESPACE
-#        define DEFINE_uint32(name, val, txt) DEFINE_VARIABLE(GFLAGS_NAMESPACE::uint32, U, name, val, txt)
-#    else
-#        define DEFINE_uint32(name, val, txt) DEFINE_VARIABLE(gflags::uint32, U, name, val, txt)
```

Comment:
can we also remove such WA from other places in repo?

Reply:
Removed